cassandra/README.md

```diff
@@ -7,343 +7,149 @@
-
-This example also has a few code and configuration files needed.  To avoid
-typing these out, you can `git clone` the Kubernetes repository to your local
-computer.
-
-## Cassandra Docker
+---
+title: Using StatefulSets with Cassandra
+---
 
+{% capture overview %}
+This tutorial shows you how to develop a native cloud [Cassandra](http://cassandra.apache.org/) deployment on Kubernetes. In this instance, a custom Cassandra `SeedProvider` enables Cassandra to discover new Cassandra nodes as they join the cluster.
+
+**Cassandra Docker**
 The pods use the [```gcr.io/google-samples/cassandra:v12```](https://github.com/kubernetes/examples/blob/master/cassandra/image/Dockerfile)
 image from Google's [container registry](https://cloud.google.com/container-registry/docs/).
-The docker is based on `debian:jessie` and includes OpenJDK 8. This image
-includes a standard Cassandra installation from the Apache Debian repo.  Through the use of environment variables you are able to change values that are inserted into the `cassandra.yaml`.
+The docker is based on `debian:jessie` and includes OpenJDK 8. This image includes a standard Cassandra installation from the Apache Debian repo.  By using environment variables you can change values that are inserted into `cassandra.yaml`.
 
 | ENV VAR       | DEFAULT VALUE  |
 | ------------- |:-------------: |
 | CASSANDRA_CLUSTER_NAME | 'Test Cluster'  |
 | CASSANDRA_NUM_TOKENS  | 32               |
 | CASSANDRA_RPC_ADDRESS | 0.0.0.0          |
 
-## Quickstart
-
-If you want to jump straight to the commands we will run,
-here are the steps:
-
-```sh
-#
-# StatefulSet
-#
+{% endcapture %}
 
-# clone the example repository
-git clone https://github.com/kubernetes/examples
-cd examples
+{% capture objectives %}
+* Create and Validate a Cassandra headless `Service`.
+* Use a StatefulSet to create a Cassandra ring.
+* Validate the StatefulSet.
+* Modify the StatefulSet.
+* Delete the StatefulSet and its pods.
+{% endcapture %}
 
-# create a service to track all cassandra statefulset nodes
-kubectl create -f cassandra/cassandra-service.yaml
-
-# create a statefulset
-kubectl create -f cassandra/cassandra-statefulset.yaml
-
-# validate the Cassandra cluster. Substitute the name of one of your pods.
-kubectl exec -ti cassandra-0 -- nodetool status
-
-# cleanup
-grace=$(kubectl get po cassandra-0 -o=jsonpath='{.spec.terminationGracePeriodSeconds}') \
-  && kubectl delete statefulset,po -l app=cassandra \
-  && echo "Sleeping $grace" \
-  && sleep $grace \
-  && kubectl delete pvc -l app=cassandra
+{% capture prerequisites %}
+To complete this tutorial, you should already have a basic familiarity with [Pods](/docs/concepts/workloads/pods/pod/), [Services](/docs/concepts/services-networking/service/), and [StatefulSets](/docs/concepts/workloads/controllers/statefulset/). In addition, you should:
 
-#
-# Resource Controller Example
-#
+* [Install and Configure](/docs/tasks/tools/install-kubectl/) the `kubectl` command line
```

Comment:
For now we should have full https://kubernetes.io/docs/... links, I am hoping to add a post-processing step to `update-imported-tutorials.py` to remove these at the time we import them.

Otherwise, these links won't work correctly.